app/models/project.rb

```diff
@@ -93,15 +95,37 @@ def category
     present_category
   end
 
+  # Because Active Storage doesn't support serving files through cloudfront (or any other way)
+  # we need to manually strip the S3 path and prepend it with the CDN url
+  def cdn_url(url)
+    regex = /.*amazonaws\.com(\/variants\/.*\/.*)\?/
+    path = regex.match(url).captures
+
+    return url if path.nil?
+
+    cdn_url = "https://dafi4qalmitzb.cloudfront.net"
```

Comment:
can you set this up as an env variable pls?

Reply:
done!